MiniProjectUIKit/Component/RecipeCollectionViewCell.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  RecipeCollectionViewCell.swift
+//  MiniProjectUIKit
+//
+//  Created by Vincent Saranang on 06/12/24.
+//
+
+import UIKit
+
+class RecipeCollectionViewCell: UICollectionViewCell {
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let areaBadge = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+        setupFrame()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
```suggestion
        super.init(coder: coder)
```
This might lead to a crash if triggered. Consider to change this code.